frappe/public/js/frappe/form/controls/table_multiselect.js

```diff
@@ -62,6 +62,7 @@ frappe.ui.form.ControlTableMultiSelect = class ControlTableMultiSelect extends f
 					[link_field.fieldname]: value
 				});
 			}
+			frappe.add_link_title(link_field.options, value, label);
```

Comment:
Undefined is being displayed for each title when this feature is used in the table multi-select field

Sider is pointing this issue and it looks related
https://sider.review/gh/repos/1864194/pulls/13848
![image](https://user-images.githubusercontent.com/28212972/131346994-3490d134-7a06-4260-8edd-5e7341a3b8a6.png)
Looks like a merge issue.
I think label has to be accepted on line 52 - https://github.com/Bloomstack/frappe/blob/production/frappe/public/js/frappe/form/controls/table_multiselect.js#L52
Please do tell me if I am missing something.
Thanks